src/api/routes/email-assignment.router.ts

```
@@ -160,10 +160,10 @@ router.post(
       action: "bulk_assigned",
       userId: req.user?.id || "system",
       details: {
-        emailCount: updatedEmails.length,
+        emailCount: updatedEmails?.length || 0,
         assignedTo: assignedTo || null,
         assigneeName,
-        errors: errors.length > 0 ? errors : undefined,
+        errors: errors?.length || 0 > 0 ? errors : undefined,
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)